DEVELOPMENT.md

```diff
@@ -0,0 +1,24 @@
+# Development Guide
+
+## Docker local development setup
+
+1. You should have docker installed in your system, if not click [here](https://docs.docker.com/get-docker/).
+
+2. Build the docker image:
+    ```shell
+    docker build -t clion/ubuntu/cpp-env:1.0 -f Dockerfile .
+    ```
+3. Set up CLion:
+    - Navigate to Settings > Build, Execution, Deployment > Toolchains
+    - Select _+_ then choose _Docker_ from the dropdown
+    - Click the gear icon on the right side of the server field
+    - Select the docker image  from the dropdown field
+      labeled _Container Settings_ and wait for CLion to
+      complete the configuration.
```

Comment:
what I did was:
- Select the docker image `clion/ubuntu/cpp-env:1.0` from the dropdown field 
(see screenshot above)
We might have difference CLion versions, thus the UI is different?

Reply:
Hey Filip, it looks like you already had a docker toolchain configuration set up on your copy of clion and that it's set up to use an already-built container called xtensa? My recommendation would be to create another docker-based Toolchain config  but rename is something unique to identify it from the previous config. I just reproduced the steps above and it should match. Let me know if that makes sense, I'd be happy to jump on a call. Cheers!